.github/labels.yml

```diff
@@ -269,9 +246,6 @@
 - name: status/waiting
   color: edb3a6
   description: "Status: Waiting"
```

Comment:
What's this one for?

Reply:
Good question, taking a deeper look, we could drop all of them but "blocked".